apps/jobs/migrations/0001_initial.py

```diff
@@ -33,9 +33,9 @@ class Migration(migrations.Migration):
                 ('started_at', models.DateTimeField(blank=True, null=True)),
```

Comment:
We never ever update the migration files. The reason behind this is many people must have already setup their database with the current master branch code. Altering migration in this way would not reflect changes in their database schema. It will only apply where a user is starting a fresh install. Any other alter can corrupt their database install, so to deal with this we always create a new migration (no matter how small the change be)